utils/ifupdown.sh.in

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Also note that bridge-stp (which is more likely to be used on an embedded system) already contains several bashisms and uses #!/bin/bash